bin/millau/runtime/src/lib.rs

```diff
@@ -234,23 +235,21 @@ impl pallet_aura::Config for Runtime {
 impl pallet_beefy::Config for Runtime {
```

Comment:
@acatangiu @svyatonik @serban300 
guys, can you please confirm this change for `pallet_beefy` pallet?

1. `MaxSetIdSessionEntries` ?
   0 or
   ```
   parameter_types! {
	pub const BeefySetIdSessionEntries: u32 = BondingDuration::get() * SessionsPerEra::get();
   }
   ```   
2. `EquivocationReportSystem = ()` ?

Reply:
Sorry - my comment was for the `pallet_grandpa`

Reply:
But I guess here it is the same. We'll probably even don't need this `pallet_session::historical` and may use `()` as `KeyOwnerProof` (if it compiles - at least it was previously)

Reply:
aha, ok, thank you, removing,
`()` does not work, but `sp_core::Void` does

Reply:
yes, `sp_core::Void` is fine for millau and rialto, we don't expect or test equivocations there :) (both beefy and grandpa)